functions/Backup-DbaDatabase.ps1

```diff
@@ -279,10 +294,19 @@ function Backup-DbaDatabase {
                     return
                 }
             }
-            if ($null -ne $AzureBaseUrl) {
-                $AzureBaseUrl = $AzureBaseUrl.Trim("/")
+            if ('' -ne $AzureBaseUrl) {
```

Comment:
A cleaner way to do this (and also check for null:

`if ( [string]::IsNullOrEmpty($AzureBaseUrl) ) {...`